lib/kube_deploy_tools/artifact_registry/driver_gcs.rb

```diff
@@ -84,6 +84,22 @@ def download_artifact(input_path, output_dir_path)
       output_path
     end
 
+    def get_latest_build_number(project)
+      out, err, status = Shellrunner.run_call('gsutil', 'ls -l', '#{@bucket}/project/#{project}/build/')
```

Comment:
I think you want to separate `ls` and `-l` into separate arguments

Reply:
maybe we could share this path building logic with `get_registry_artifact_path`? since it's a pretty critical piece of functionality (if this name changes, everything breaks), I think it would be a good idea to do it only in one place

Reply:
agreed with Romain. Other ideas:

* consider using `check_call` to avoid needing to put in the error check
* why use `ls -l` when `ls` would do? I don't think you need the extra long listing information, you just need the names of the prefixes / directories and plain ls gives you that.